app/models/mappers/reed_mappers/job_mapper.rb

```diff
@@ -23,9 +23,10 @@ def initialize(data)
           @data = data
         end
 
-        def build_entity
+        def build_entity # rubocop:disable Metrics/MethodLength
           Entity::Job.new(
-            id: @data['jobId'].to_s,
```

Comment:
If this can be `nil`, then I'm not sure why we need this field?

Reply:
it will be used to [keep the database id](https://github.com/KeroKnows/s-killer/pull/13/files#diff-237c3ef51bacc6b008b21b89cf4769f7558547d7bce199f99dfd6934c7422467R45)

Reply:
Gotcha